src/client/index.js

```diff
@@ -132,6 +132,11 @@ class Hammerhead {
 
 var hammerhead = new Hammerhead();
 
-window.Hammerhead = hammerhead;
+Object.defineProperty(window, '@hammerhead@', {
```

Comment:
I don't think this exact var will ever appear in the URL. So, it's completely OK. Moreover, we will need to modify testcafe vars due to the incosistency. `%`-notation for modules is also better because it will make it distinct from our internal constants, which is good.